### DISCUSSION TITLE
Welcome to openhab-google-assistant Discussions!

### DISCUSSION THREAD

**Discussion by michikrug** [openhab/openhab-google-assistant#299] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SeveranExp** [comment]
Hi! 
Thank you for making this space to allow some discussion on parallel topics! Sometimes it’s not clear whether or not something is an issue, and I’ll be taking advantage of this space to ask a question in just a second :)
I’m Sev, or Pedro, whatever name you find more confortable, I’m from Portugal and I’ve been using openHAB since way back in the v2. 
Most of my knowledge was self acquired so I definitely lack some key insights (which result in me asking stupid questions.) so sorry for that :)
Looking forward to collaborating with others!

edit: actually, I can’t create new topics :/

**Reply by michikrug** [reply]
Could it be that you tried creating a discussion in the "Announcement" category? I believe this is reserved for maintainers.
Can you try again maybe in "General" category?

**Reply by SeveranExp** [reply]
I tried again in the general like you asked and this is what I see:
![IMG_5801](https://github.com/openhab/openhab-google-assistant/assets/30979860/0a1bf5f6-4add-445f-9bb7-75a7f69f9491)

I don’t see a button to create new topics anywhere. Am I in the correct place?

**Reply by michikrug** [reply]
Yes, that is the right place. There should be a green button next to filter.

![Screenshot 2023-05-24 at 15 55 41](https://github.com/openhab/openhab-google-assistant/assets/727517/5041cc63-3a2c-4220-8cb7-b4dc208fe4c7)

Also the documentation states:
> Any authenticated user who can view the repository can create a discussion in that repository. 

I am a bit lost why this is not the case for you...

It might be the case that there is some organisation (openhab) setting out of my control that restricts this.

**Reply by michikrug** [reply]
@kaikreuzer do we limit starting discussions on the organisation level? if so would be open to change this?

Reference: https://github.com/orgs/community/discussions/29#discussioncomment-112122
